### PR TITLE
Add binding for '⌥+Tab' to '⌘+Tab'

### PR DESCRIPTION
**Pull request by rustycl0ck**

### CODE REVIEW

**Comment by MuhammedZakir** [comment]
As yours is pure json, there is no need for `.erb` file. Also, use `left_command` in place of `left_gui`.

**Comment by rustycl0ck** [comment]
Thanks for the comments. I have removed the `.erb` file and switched from `left_gui` to `left_command`. Also added an extra description file.

File: public/json/option-tab_to_command-tab.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+              ]
+            }
+          },
+          "to": [
+            {
+              "key_code": "tab",
+              "modifiers": [
+                "left_command"
+              ],
+              "lazy": true
```
Comment (MuhammedZakir) [inline public/json/option-tab_to_command-tab.json:25]:
```suggestion
              ]
```

AFAIK, `to.lazy` only works if `from.keycode` is a modifier key.

From https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to/lazy/:
> If `to.lazy` is `true` and `to.key_code` is modifier flag such as `"key_code": "left_shift"`, the `to.key_code` acts as lazy modifier.

File: public/extra_descriptions/option-tab_to_command-tab.json.html [hunk 2]
```diff
@@ -0,0 +1 @@
+Use '⌥+Tab' to cycle through applications (in addition to the default '⌘+Tab')
```
Comment (MuhammedZakir) [inline public/extra_descriptions/option-tab_to_command-tab.json.html:1]:
```suggestion
<link rel="stylesheet" href="../../vendor/css/bootstrap.min.css" />

<p>Use '⌥+Tab' to cycle through applications (in addition to the default '⌘+Tab')</p>
```

**Review by MuhammedZakir — CHANGES_REQUESTED** [review]

**Comment by MuhammedZakir** [comment]
I will merge this once those two are solved. Thanks! :-)

File: public/json/option-tab_to_command-tab.json (hunk 1 above)
Comment (rustycl0ck) [reply public/json/option-tab_to_command-tab.json:25]: Thanks for pointing this out. I didn't fully understand the function of `lazy`. I have removed it now.

File: public/extra_descriptions/option-tab_to_command-tab.json.html (hunk 2 above)
Comment (rustycl0ck) [reply public/extra_descriptions/option-tab_to_command-tab.json.html:1]: Thanks for suggesting this. I have updated it as requested.

**Comment by MuhammedZakir** [comment]
Merged. :-)

**Pull request merged**